client/src/resources/lang/en_us.json

```diff
@@ -937,6 +944,11 @@
     "wiki.article.role.auditor.attributes":"- You are not told who the player with the role is\n- The random role you are told is from the outline you chose",
     "wiki.article.role.auditor.extra":"- Your ability works independently of your visits\n    - Your visit getting possessed or transported doesn't impact your ability\n    - If you visit a player in jail, you aren't told they're in jail, and you still get results\n    - If you are roleblocked, you don't get results, but you can select that outline again\n- It is possible for you to visit dead players",
 
+    "wiki.article.role.snoop.guide": "- At night, select a player, you are either told that they are a townie, or that you can't tell if they are a townie\n    - You can't tell what your target was if:\n        - They aren't a townie\n        - You were visited by anyone at all\n        - They have a suspicious aura\n\n    - Otherwise, you are told they are a townie",
+    "wiki.article.role.snoop.abilities":"- Visit a player to search them\n    - If your target is a townie, and you are not visited by anyone, you are told they are a townie",
+    "wiki.article.role.snoop.attributes":"- If your target has a suspicious auras, your results will be inconclusive",
+    "wiki.article.role.snoop.extra":"- You are still told if you are roleblocked, or if your target was in jail",

```

Comment:
still told this when? You should clarify that you mean that this means 'even when visited'.